src/models/room/RoomRequestBody.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  validateExpiredAt,
+  validateId,
+  validateLongBreak,
+  validateLongBreakInterval,
+  validateMasterId,
+  validatePassword,
+  validateShortBreak,
+  validateThumbnail,
+  validateTimer,
+  validateTitle,
+} from "@/utils/rooms.validator";
+import { Room } from "@/stores/RoomListStore";
+~21;
+export class RoomRequestBody {
+  private room: Room;
```

Comment:
private은 언더바로 시작하기로 했어서 컨벤션을 지켜주심 됩니다.
```suggestion
  private _room: Room;
```